### DISCUSSION TITLE
How to run AppImage files

### DISCUSSION THREAD

**Discussion by og900aero** [sxyazi/yazi#651] [1. Q&A] [answered]
I want to run/open AppImage file, but stuck o task. I try the following codes in yazi.toml

```
[opener]

app_image = [
	{ exec = 'bash -c "$1"', desc = "Run AppImage", for = "unix" }
]
```

or 

```
app_image = [
	{ exec = '"$1"', desc = "Run AppImage", for = "unix" }
]
```

**Comment by SolitudeSF** [comment]
```
app_image = [
	{ exec = '"./$1"', desc = "Run AppImage", for = "unix" }
]
```
does this work?

**Reply by og900aero** [reply]
I tried it too, but doesnt work.

**Comment by sxyazi** [comment]
Try
```
app_image = [
	{ exec = 'xdg-open "$1"', desc = "Run AppImage", for = "unix" }
]
```

**Reply by og900aero** [reply]
Not working properly. This method run Chromium, and downloaded file :D

**Reply by sxyazi** [reply]
How do you run it from your terminal? It should be the same way

**Comment by og900aero** [comment, marked as answer]
> app_image = [
> 	{ exec = '"$1"', desc = "Run AppImage", for = "unix" }
> ]

Unfortunately not, because then this code would be good, but it still gets stuck in the task:

```
app_image = [
	{ exec = '"$1"', desc = "Run AppImage", for = "unix" }
]
```
After many attempts, the solution was:

```
app_image = [
	{ exec = '$TERM -e "$1"', orphan = true, desc = "Run AppImage", for = "unix" }
]
```
